community/modules/scripts/ramble-setup/outputs.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "ramble_setup_runner" {
```

Comment:
In spack design we decided to call this the equivalent of "ramble_runner" to making chaining easier. Do we want to do that here? The name is less clear but then it can be consistent between different modules.

Reply:
Good catch. Done.